contracts/TickerRegistry.sol

```diff
@@ -91,11 +107,29 @@ contract TickerRegistry is ITickerRegistry, Util, Pausable, RegistryUpdater, Rec
         require(msg.sender == securityTokenRegistry, "msg.sender should be SecurityTokenRegistry contract");
         require(registeredSymbols[symbol].status != true, "Symbol status should not equal to true");
         require(registeredSymbols[symbol].owner == _owner, "Owner of the symbol should matched with the requested issuer address");
-        require(registeredSymbols[symbol].timestamp.add(expiryLimit) >= now, "Ticker should not be expired");
+        require(registeredSymbols[symbol].expiredTimestamp >= now, "Ticker should not be expired");
         registeredSymbols[symbol].tokenName = _tokenName;
         registeredSymbols[symbol].status = true;
         return true;
     }
+   
+    /**
+     * @notice Register the ticker without paying the fee 
+       Once the token symbol is registered to its owner then no other issuer can claim
+       Its ownership. If the symbol expires and its issuer hasn't used it, then someone else can take it.
+     * @param _symbol token symbol
+     * @param _tokenName Name of the token
+     * @param _swarmHash Off-chain details of the issuer and token
+     */
+    function addCustomTicker(string _symbol, string _tokenName, bytes32 _swarmHash) public onlyOwner {
```

Comment:
addCustomTicker should allow us to fully recreate the ticker if needed, so probably it should accept every single parameter as it comes.